Zend/zend_builtin_functions.c

```diff
@@ -1969,7 +1969,14 @@ ZEND_API void zend_fetch_debug_backtrace(zval *return_value, int skip_last, int
 
 	while (ptr && (limit == 0 || frameno < limit)) {
```

Comment:
Instead of using `limit=0` to mean "unlimited", can we please use `INT_MAX`? It will simplify the code here:
```
	while (ptr && frameno < limit) {
```
We can add a constant for it if it helps comprehension.

Aside from simplifying code I much prefer it when `0` isn't used for "infinite" or "max", because as we `--limit` the stack length with get smaller and smaller until we hit `0`, and then magically it gets bigger again? `INT_MAX` is a much better fit, IMO.

Reply:
@morrisonlevi thanks for comments on this. I'll consider it if I start with this implementation as a separate PECL extension.